test/clj/rems/test_browser.clj

```diff
@@ -786,3 +787,52 @@
               "Active" ""}
              (slurp-fields :workflow)))
       (is (btu/visible? {:tag :a :fn/text "Simple form"})))))
+
+(deftest test-blacklist
+  (btu/with-postmortem {:dir btu/reporting-dir}
+    (testing "set up resource & user"
+      (test-data/create-resource! {:resource-ext-id "blacklist-test"})
+      (users/add-user! "baddie" {:eppn "baddie" :commonName "Bruce Baddie" :mail "bruce@example.com"}))
+    (testing "add blacklist entry via resource page"
+      (login-as "owner")
+      (go-to-admin "Resources")
+      (click-row-action [:resources] {:fn/text "blacklist-test"}
+                        (select-button-by-label "View"))
+      (btu/wait-visible {:tag :h1 :fn/text "Resource"})
+      (btu/wait-page-loaded)
+      (is (= [{}] (slurp-rows :blacklist)))
+      (btu/fill-human :blacklist-user "baddie\n")
+      (btu/fill-human :blacklist-comment "This is a test.")
+      (btu/screenshot (io/file btu/reporting-dir "test-blacklist-1.png"))
+      (btu/scroll-and-click :blacklist-add)
+      (btu/wait-visible {:css ".alert-success"})
+      (is (str/includes? (btu/get-element-text {:css ".alert-success"}) "Success")))
+    (testing "check entry on resource page"
+      (is (= [{}
```

Comment:
Is this the header?

Reply:
yeah, I think so

Reply:
Perhaps some of the other tables us `thead` and `tbody` to avoid having to remove the header row. Slurping could support removing the header anyway.

Reply:
Can you add a TODO for it? I think we can merge this one anyway.

Reply:
yeah